src/AskChatGPTModal.tsx

```diff
@@ -42,8 +42,8 @@ export class AskChatGPTModal extends Modal {
 	}
 
 	async saveToAndOpenNewNote(text: string) {
-		const noteRandomId = Math.floor(Math.random() * (100000 - 1) + 1);
-		const newNote = await this.app.vault.create(`/vaultchat-${noteRandomId}.md`, text)
+		const dateTime = (new Date().toDateString()).split(" ").join("-")
```

Comment:
What does this end up looking like? Is it just the date or does it include the time? If it's just the date then you would only be able to save one chat per day without collisions.

Reply:
Ran in dev console and got this:
```
> new Date().toDateString().split(" ").join("-")
< 'Wed-Mar-15-2023'
```

I think we want something more like `Vault Chat - 2023-03-14 09:37:21.22.md`